### PR TITLE
LPD-23145 

### PR DESCRIPTION
**Pull request by igor-franca**
Previous [PR](https://github.com/liferay-objects/liferay-portal/pull/3241)

Related Tickets:
https://liferay.atlassian.net/browse/LPD-23146
https://liferay.atlassian.net/browse/LPD-23145

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ModelBuilderContext/objectFolderReducer.ts [hunk 1]
```diff
@@ -843,6 +843,37 @@ export function ObjectFolderReducer(state: TState, action: TAction): TState {
 			};
 		}
 
+		case TYPES.SET_SELECTED_OBJECT_FOLDER_DETAILS: {
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ModelBuilderContext/objectFolderReducer.ts:846]: Hey @igor-franca, could you sort this function alphabetically?

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @igor-franca, Good job, I have only one suggestion :smile:

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by igor-franca** [comment]
ci:forward

**Pull request closed without merging**